tests/group/test_group.yml

```diff
@@ -174,6 +174,129 @@
     register: result
     failed_when: result.changed or result.failed
 
+  - name: Ensure group group1 is absent
```

Comment:
It should be "Ensure group group3, group2 and group1 are absent".

Reply:
Fixed.